### PR TITLE
[2.2.x] Refs #31485 -- Backported jQuery upgrade to 3.5.1.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

**Comment by carltongibson** [comment]
CI failures: 

* flake8: new flake8 version
* Docs: _WARNING: term feature release not found in case sensitive match.made a reference to Feature release instead._

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -2149,6 +2149,10 @@ in your own admin JavaScript without including a second copy, you can use the
 
     jQuery was upgraded from 2.2.3 to 3.3.1.
 
+.. versionchanged:: 2.2
```
Comment (felixxm) [inline docs/ref/contrib/admin/index.txt:2152]: `2.2.13`

**Comment by carltongibson** [comment]
Thanks @felixxm. Updated.

**Comment by carltongibson** [comment]
buildbot, test on selenium.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Pull request merged**